spec/models/programme_spec.rb

```diff
@@ -318,4 +318,16 @@
       expect { generic_programme.short_name }.to raise_error(NotImplementedError)
     end
   end
+
+  describe '#mailer' do
```

Comment:
These were just some missing tests in some of the files that I altered so I've added them in here to take it up to 100% coverage